test/unit/models/edition_taxonomy_tag_form_test.rb

```diff
@@ -18,17 +20,45 @@ class EditionTaxonomyTagFormTest < ActiveSupport::TestCase
     form = EditionTaxonomyTagForm.load(content_id)
 
     assert_equal(form.edition_content_id, content_id)
-    assert_equal(form.taxons, taxons)
+    assert_equal(form.selected_taxons, taxons)
     assert_equal(form.previous_version, 1)
   end
 
-  test '#publish should send a patch links to publishing-api' do
+  test '#publish should send a patch links to publishing-api with the most specific selected taxons' do
```

Comment:
That is the intention, to have a structure that looks like the education taxonomy. You will need to know what's in the `EducationTaxonomyHelper` because that's how it looks from PublishingAPI. The reason `EducationTaxonomyHelper` exists is because we have other tests that need the stub. I don't think we should change this at this point.